pageserver/src/tenant/timeline.rs

```diff
@@ -1598,7 +1598,9 @@ impl Timeline {
     /// Initialize with an empty layer map. Used when creating a new timeline.
     ///
     pub(super) fn init_empty_layer_map(&self, start_lsn: Lsn) {
```

Comment:
shouldn't this be an optional parameter to ```create_timeline_struct``` that will ensure the initialization of the timeline with an empty layer ?